SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security
+
+Reporting process and disclosure communications are outlined in [/org/security](https://github.com/opencontainers/org/blob/master/security/)
```

Comment:
Nits for consideration, add "The" to front and period '.'at end.
```
The reporting process and disclosure communications are outlined in [/org/security](https://github.com/opencontainers/org/blob/master/security/).
```

Reply:
Thanks, applied to README as well.